sdk/core/core-rest-pipeline/documentation/pipeline.md

```diff
@@ -0,0 +1,220 @@
+# Policies
+
+A pipeline policy manipulates a request as it travels through the pipeline. It is conceptually a middleware that is allowed to modify the request before it is made as well as the response when it is received.
```

Comment:
should we also mention the bubbling nature here? i.e. the first policy to touch the request will be the last to modify the response?